config/application.rb

```diff
@@ -37,6 +35,5 @@ class Application < Rails::Application
     #
     # config.time_zone = "Central Time (US & Canada)"
     # config.eager_load_paths << Rails.root.join("extras")
-    ActiveSupport.escape_html_entities_in_json = false
```

Comment:
This feels slightly surprisingly that it would be in user_html, as its scope is wider than that, isn't it?

Reply:
Well, I moved it to a new initializer, html_in_json.rb